src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
-body {
+$days:
+  "mon" 0,
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
+$month-length: 28, 29, 30, 31;
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  padding: 10px;
+  margin: 500px auto;
+  max-width: 800px;
+
+  &__day {
+    position: relative;
+
+    width: 100px;
+    height: 100px;
+
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &__day:nth-child(#{$index})::before {
+      position: absolute;
+      top: 35px;
+      left: 35px;
```

Comment:
I think this is a bad idea. You need to use display:flex to align items center.

Reply:
You have a right. Fixed according to a suggestion